### ISSUE TITLE
Timers on website depend on your computers system clock

### ISSUE THREAD

**Issue by sdfgeoff** [ludumdare/ludumdare#716]
I flew from GMT+12 to GMT+1, and did not update my system clock. According to the timers on the website, the jam had already started! Fiddling around with my system clock/timezone settings means I can twiddle the website's timers to, well, wherever I like.

While it's not a serious problem (everyone's computer clock is probably close enough to being right), it may result in people missing the submission deadline if they rely only on the websites timers.

**Comment by zwrawr** [comment]
The site should check the server time against the deadline as well as the client time against the dead line, and tell the user something might be wrong if there discrepant.

**Comment by povrazor** [comment]
Yep, I added a server time variable to one of the main files that can be used for this.
